viz-lib/src/visualizations/table/renderer.less

```diff
@@ -12,6 +12,20 @@
     overflow-x: auto;
   }
 
+  .table-fixed-header {
+    table {
+      border-top: 0;
+
+      th {
+        position: sticky !important;
+        left: 0;
+        top: 0;
+        border-top: 1px solid #e8e8e8;
```

Comment:
this line code should change `border-top: 0px solid #e8e8e8;` , `1px` will result in 1px gap.

Reply:
Weird, I've added this exactly to replace the Table-level border. This is how it looks to me in a dashboard:

![dashboard-border](https://user-images.githubusercontent.com/3356951/90352508-feb06a00-e019-11ea-86d6-09d935d496fa.gif)

Which browser are you using?

Reply:
Maybe it's my version problem, it worker after i change 1px to 0px or hidden. Don't worry, it already works. Thank you.

Reply:
> Weird, I've added this exactly to replace the Table-level border. This is how it looks to me in a dashboard:
> 
> ![dashboard-border](https://user-images.githubusercontent.com/3356951/90352508-feb06a00-e019-11ea-86d6-09d935d496fa.gif)
> 
> Which browser are you using?

I use Chrome.

Reply:
I deleted a few Chinese citation comments so that more people can use PR.